main.js

```diff
@@ -1 +1,76 @@
-main.js
\ No newline at end of file
+'use strict'
+
+{
+
+    //追加ボタンを押下時にタスクを登録できる
+
+    const add = document.getElementById('add');
+    const input =document.getElementById('input');
+    const tbodyTasks = document.getElementById('tasks');
+    const todos = [];
+    const btn_remove = document.createElement('button');
```

Comment:
JSではキャメルケースを使用しましょう。
```suggestion
    const btnRemove = document.createElement('button');
```